stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_composer_default_leading_content.xml

```diff
@@ -1,37 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    >
+<merge xmlns:android="http://schemas.android.com/apk/res/android">
 
     <androidx.appcompat.widget.LinearLayoutCompat
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:gravity="center"
+        android:gravity="bottom"
```

Comment:
I'm for keeping controls at the bottom in case of multiline input as:

- It seems to be like that in our designs
- I don't like controls running away from me when typing multiline message.